test/sql/test_functions.py

```diff
@@ -215,6 +216,33 @@ def test_generic_random(self):
         ]:
             self.assert_compile(func.random(), ret, dialect=dialect)
 
+    def test_generic_string_agg(self):
+        t = table("t", column("value", String))
+        expr = func.string_agg(t.c.value, ",")
+        is_(expr.type._type_affinity, String)
+        stmt = select(expr)
+
+        self.assert_compile(
+            stmt,
+            "SELECT group_concat(t.value, ?) AS string_agg_1 FROM t",
+            dialect=sqlite.dialect(),
```

Comment:
it may make sense to add a test (maybe in dialects/test_sqlite) using a single arg

Reply:
I added some compilation tests with a single arg still in the same `test/sql/test_functions.py` file, because there are other tests like these here and because I couldn't see a suitable place in `test_sqlite`, but let me know if you think these should be somewhere else

Reply:
Should be ok